docker/Dockerfile.cpu

```diff
@@ -20,12 +20,7 @@ RUN apt-get update && \
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
 
-RUN pip install --no-cache-dir psutil
-RUN pip install --no-cache-dir torch
-RUN pip install --no-cache-dir torchvision
-RUN pip install --no-cache-dir torchtext
-ADD serve serve
-RUN pip install ../serve/
+RUN pip install --no-cache-dir psutil future torch torchvision torchtext torchserve
```

Comment:
Can't we have a separate `requirements.txt` in docker folder with all the required packages. That way we can simply `COPY requirements.txt` in the docker file and `RUN pip install -r requirements.txt` in all the Dockerfiles. We will have one place to update if we need to add packages or pin the versions of packages right?

Reply:
@vdantu @chauhang I have added you both in this PR for a separate requirements.txt
Please review and comment there as well. Thanks.
https://github.com/pytorch/serve/pull/228

Reply:
#228 will address the requirement.txt and new issue can be created to incorporate those changes in new Dockerfile. Please suggest.